### DISCUSSION TITLE
Add support for Sapling SCM

### DISCUSSION THREAD

**Discussion by osiewicz** [zed-industries/zed#45714] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

The need for support of [Sapling](https://github.com/facebook/sapling) SCM came up when I talked to @davidbarsky 

### Environment

Zed: v0.165.0 (Zed Nightly 196fd65601af0b22cba3f6f0fbbda821b0403427)
OS: macOS 15.1.0
Memory: 64 GiB
Architecture: aarch64

**Comment by JosephTLyons** [comment]
Just going to label this as `git`, even though that is not quite correct, to avoid making a new label.

**Comment by jansol** [comment]
Does this mean that other SCMs are on the table as well (to open tracking issues for) or do you still intend to limit support to git-compatible ones? Subversion is still being used by a number of projects, personally I'm interested in pijul and I know at least darcs having a solid following. And then there's of course Perforce in the corporate space...
